contrib/ansible/group_vars/all.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: cluster.local
 # Account name of remote user. Ansible will use this user account to ssh into
 # the managed machines. The user must be able to use sudo without asking
 # for password unless ansible_sudo_pass is set
-#ansible_ssh_user: root
+ansible_ssh_user: fedora
```

Comment:
alas : Vagrant doesnt support vars in inventory,since its bad practice.  So i geuss sed replace is the only way to do it (my ansible refuses to override parameters all.yml, i suspect others might have this problem)